ArduPlane/events.cpp

```diff
@@ -234,13 +233,13 @@ void Plane::failsafe_long_on_event(enum failsafe_state fstype, ModeReason reason
 #endif
         break;
     }
-    gcs().send_text(MAV_SEVERITY_WARNING, "%s Failsafe On: %s", (reason == ModeReason:: GCS_FAILSAFE) ? "GCS" : "RC Long", control_mode->name());
+    gcs().send_text(MAV_SEVERITY_WARNING, "%s Failsafe On: switched to %s", (reason == ModeReason:: GCS_FAILSAFE) ? "GCS" : "RC Long", control_mode->name());
 }
 
 void Plane::failsafe_short_off_event(ModeReason reason)
```

Comment:
I think it would make this easier to understand if the function name made it clear that it is only RC fail safes. 
eg:
`rc_failsafe_short_off_event`

Reply:
short failsafes are only a plane rc thing, not sure it would be confused with anything else...but name change does not hurt anything

Reply:
oops, indeed it does....my mistake